BusyBooth/BusyBooth.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,8 @@
 		6B7DF8851C6F77A100D43104 /* Constants.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B7DF8841C6F77A100D43104 /* Constants.m */; };
 		6B7DF8891C6FAA0400D43104 /* PollingPlaceViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B7DF8881C6FAA0400D43104 /* PollingPlaceViewController.m */; };
 		6B7DF88C1C6FAD2700D43104 /* PastPollingViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B7DF88B1C6FAD2700D43104 /* PastPollingViewController.m */; };
+		6B8AE3581C7829C6003E51BC /* SettingsTableViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B8AE3571C7829C6003E51BC /* SettingsTableViewController.m */; };
+		6B8AE35C1C7832A8003E51BC /* MyAccountViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B8AE35B1C7832A8003E51BC /* MyAccountViewController.m */; };
 		6B8DAC391C68792700AA6ED3 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 6B8DAC381C68792700AA6ED3 /* main.m */; };
```

Comment:
Any idea what the changes in this file are? I thought that you're supposed to .gitignore the pbxproj